### ISSUE TITLE
Bursa aspirate: WBC count >10,000 / mm3

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10818]
**Preferred term label:**
Bursa aspirate: WBC count >10,000 / mm3

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Prepatellar bursitis
Olecranon bursitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
there is currently a descriptive term [Septic arthritis HP:0003095](https://hpo.jax.org/browse/term/HP:0003095)
Is it really a good idea to limit this to >10,000 cells?
According to this source (https://emedicine.medscape.com/article/309014-workup?form=fpf) septic bursitis is characteirzed by 1500-300,000; mean 75,000
Possibly it is better to have the definition be "above the upper limit of normal" and to add typical ranges in the comment?

**Comment by MickeySegal** [comment]
Bursitis and synovitis are different: 
https://www.towsonortho.com/is-my-joint-pain-arthritis-or-bursitis/
https://www.abdn.ac.uk/medical/elf/courses/view/135965/bursitis-and-septic-arthritis/1/page16

If anything it should be [HP:0025232](https://hpo.jax.org/browse/term/HP:0025232)

**Comment by pnrobinson** [comment]
@MickeySegal 
I added a parent term Abnormal bursal fluid morphology, and then
HP:0020256
Elevated bursal fluid leukocyte count
Def: The count of leukocytes per unit volume in a bursa aspirate specimen is above the uppper limit of normal.
Comment: This finding is characteristic of septic arthritis. To ascertain this finding, typically, a specimen of bursal fluid is removed from the inflammed joint with a needle and syringe (aspiration) and examined microscopically.

I found many different cutoff values for what one expects with bursitis and do not think that 10,000 is a generally accepted value, so it is better for the HPO term to just have "above normal".
